asterisk_mbox_server/cdr.py

```diff
@@ -95,9 +95,9 @@ def _read_cdr_file(self):
         with open(self._cdr_file) as filp:
             lines = filp.readlines()
             keys = None
-            entries = csv.reader(lines, quotechar='"', delimiter=',',
+            entries = list(csv.reader(lines, quotechar='"', delimiter=',',
```

Comment:
why is this needed.  csv.reader should be iterable

Reply:
It should be iterable, but it wasn't for my version of python.